analysis/local_analog_calibration.py

```diff
@@ -110,7 +250,7 @@ def main(lR23, lO32, OH, out_pdf, ID=[], lR23_err=[], lO32_err=[],
         ax[0].scatter(lR23[nn], OH[nn], color=ctype[nn], edgecolor='none',
                       marker=marker[nn], alpha=0.5, label=label[nn])
         if len(ID) != 0:
-            for ii in range(len(lR23[nn])):
+            for ii in range(len(lR23[nn]-2)):
```

Comment:
Why are we excluding the last two points?

Reply:
I excluded the last two arrays in lR23 for including IDs because these are the IDs of the DEEP2 and MACT samples. Same case for the other instance.

Reply:
OK, but embedding it in the code is only going to leave it off permanently and if we find ourselves plotting other stacks, those will automatically be excluded. Not sure how to fix this.